dump_zenodo_open_metadata.py

```diff
@@ -53,53 +53,65 @@ def parse_record(rec, spam=False):
     out["spam"] = spam
     return out
 
+print("What type of dump you want? 1- Ham, 2- Spam or 3- Both?")
+type = input()
+
+if(type == 1):
+    type_string = "ham_"
+elif(type == 2):
+    type_string = "spam_"
+else:
+    type_string = "ham&spam_"
```

Comment:
```suggestion
DUMP_TYPES = {
  '1': 'ham',
  '2': 'spam',
  '3': 'full',
}

choices = [k + ' - ' + v for k, v in DUMP_TYPES.items()]
choice = input("What type of dump you want? {}'.format(', '.join(choices)).
dump_type = DUMP_TYPES.get(choice)
if not dump_type:
    raise Exception('Invalid dump type {}'.format(choice))
```

- Just to avoid the multi if-else block, and make it easier for future additions.
- Note also that I used strings for the keys (i.e. `"1"`), since `input()` will return a string